Apps/Inc/SendCarCAN.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __SENDCARCAN_H
+#define __SENDCARCAN_H
+
+#include "CANbus.h"
+
+/**
+ * @brief Initialize SendCarCAN
+*/
+void SendCarCAN_Init();
+
+/**
+ * @brief Wrapper to put new message in the CAN queue
+*/
+void SendCarCAN_Put(CANDATA_t message);
+
+/**
+ * @brief return the space left in SendCarCAN_Q for debug purposes
+*/
+uint8_t get_SendCarCAN_Q_Space(void);
```

Comment:
Do we need this anymore? If no, can we remove it? If yes, can we surround it with a DEBUG ifdef?

Reply:
I haven't worked/debugged this PR before, but I like the idea of keeping this function just in case 👍